spec/cart_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'cart'
+require 'dish'
+require 'viewer'
+require 'menu'
+
+meat_feast = Dish.new("Meat Feast", 950)
+four_seasons = Dish.new("Four Seasons", 650)
+veggie = Dish.new("Veggie", 700)
+margherita = Dish.new("Margherita", 500)
+
+cart = Cart.new
+
+cart.receive_item(meat_feast)
+cart.receive_item(four_seasons)
+cart.receive_item(veggie)
+
+describe Cart do
+  describe '@order' do
+    it 'can hold the order of the customer' do
+      expect(cart.order).to eq [meat_feast, four_seasons, veggie]
+    end
+  end
+
+  describe '#receive_item' do
+    before(:each) do
```

Comment:
I see you've removed some duplication here using the before block.  It's fine here but if you had another 5 test cases for the method, the before block would be a little way away from where the tests are executed and that indirection could be costly.  I'm not saying don't do it – just highlighting the trade off.  Some people will be OK with a certain amount of repetition in the tests if it brings greater clarity.